manifests/init.pp

```diff
@@ -61,6 +61,9 @@
 #
 # $qpid_router_logging_path::           Directory for dispatch router logs
 #
+# $docker_port::                        Docker registry port, 5000 (default) or 5001
```

Comment:
I think I'd prefer the clearer ```docker_registry_port```. Also the or 5001 implies you can only use 5000 or 5001. I'd just remove mentioning the values.